main.py

```diff
@@ -0,0 +1,238 @@
+import requests
+from bs4 import BeautifulSoup
+import csv
+import os
+import shutil
+import copy
+import urllib.request
+import pdb; 
+#pdb.set_trace()
+
+
+list_book_from_category = {}
+category_name_link = {}
+
+list_category = []
+category_link = []
+db_for_csv = {}
+
+db = {	 'product_page_url':'product_page_url'
+		,'universal_product_code':'UPC'
+		,'title':'title'
+		,'price_including_tax':'Price (incl. tax)'
+		,'price_excluding_tax':'Price (excl. tax)'
+		,'number_available':'Availability'
+		,'product_description':''
+		,'category':''
+		,'review_rating':'Number of reviews'
+		,'image_url':'image_url'
+		}
+
+# Create directory for output:
+def create_output_directory(p_name):
```

Comment:
Ici tu as juste levé l'exception `FileExistsError`. Mais que se passe t-il sisi tu une autre exception qui est levée ? Ton programme va se planter. Alors je te propose de lever aussi `Exception` dans un cas général et d'arrêter le programme.

Reply:
comme ceci ?
import sys

# Create directory for output:
def create_output_directory(p_name):
	try:
		os.mkdir('./' + p_name)
	except FileExistsError:
		pass
	except Exception:
		sys.exit("Un problème est survenue, veuillez contacter l'administrateur")

Reply:
oui